packages/devp2p/src/rlpx/rlpx.ts

```diff
@@ -50,6 +51,8 @@ export class RLPx {
   protected _refillIntervalId: NodeJS.Timeout
   protected _refillIntervalSelectionCounter: number = 0
 
+  protected _keccakFunction: (msg: Uint8Array) => Uint8Array
```

Comment:
Not sure, why is this added? It's used nowhere?

Reply:
Not sure, but I'll sure use it in future devp2p PRs.